pkg/operator/workload/sync_openshift_oauth_apiserver_test.go

```diff
@@ -143,59 +143,6 @@ var emptyAPIServerArgsJSON = `
 }
 `
 
-var withDefaultsProvidedAPIServerArgsJSON = `
-{
-  "oauthAPIServer": {
-    "apiServerArguments": {
-      "api-audiences": "https://now.something.different",
-      "audit-policy-file": "/var/run/configmaps/audit/writerequestbodies.yaml"
-    }
-  }
-}
-`
-
-func TestGetStructuredConfigWithDefaultValue(t *testing.T) {
-	scenarios := []struct {
-		name                       string
-		spec                       operatorv1.OperatorSpec
-		expectedAPIServerArguments map[string][]string
-	}{
-		{
-			name: "default values are assigned when not provided",
-			spec: operatorv1.OperatorSpec{
-				ObservedConfig: runtime.RawExtension{Raw: []byte(emptyAPIServerArgsJSON)},
-			},
-			expectedAPIServerArguments: map[string][]string{
-				"audit-policy-file": {"/var/run/configmaps/audit/default.yaml"},
-			},
-		},
-
-		{
-			name: "default values are NOT assigned when provided",
-			spec: operatorv1.OperatorSpec{
-				ObservedConfig: runtime.RawExtension{Raw: []byte(withDefaultsProvidedAPIServerArgsJSON)},
-			},
-			expectedAPIServerArguments: map[string][]string{
-				"api-audiences":     {"https://now.something.different"},
```

Comment:
can we preserve this part of the test about api-audiences?

Reply:
just leave the test, but remove the lines with audit policy. That should work, shouldn't it?

Reply:
I can try it.